client/app/services/axios.js

```diff
@@ -1,26 +1,36 @@
 import axiosLib from "axios";
 import { Auth } from "@/services/auth";
 import qs from "query-string";
-import Cookies from "js-cookie";
+import createAuthRefreshInterceptor from "axios-auth-refresh";
 
 export const axios = axiosLib.create({
   paramsSerializer: params => qs.stringify(params),
+  xsrfCookieName: "csrf_token",
+  xsrfHeaderName: "X-CSRF-TOKEN",
```

Comment:
Worth adding some config around this to disable if needed.

Reply:
If the cookie is absent, the header is not added.

https://github.com/axios/axios/blob/master/lib/adapters/xhr.js#L119